### DISCUSSION TITLE
Noise reduction

### DISCUSSION THREAD

**Discussion by lexCrr** [supermerill/SuperSlicer#2021] [Q&A]
Hi, 
i recently switched from Cura 4.11 to Superslicer 2.3.56 as it seemed a really good alternative. Everything is working perfectly and my models have came out great ( I'm using a sligthly upgraded ANET A6). The only problem I have this far is about noise: printing with G-code from Superslicer is an absolute horror for the ears, and my printer ( I have rubber feet to try to cancel vibrations) is usually quite quiet (not silent), but it just keeps emitting scratching noises with superslicer. I haven't seen anyone with the same problem this far and was wondering if I'm the only one or if someone has any solutions?
Thanks in advance

**Comment by supermerill** [comment]
maybe the printer limits are set to "emit limits to gcode" and these limits (acceleration & jerk) are too high.

**Comment by lexCrr** [comment]
<img width="956" alt="Capture d’écran 2021-12-08 203106" src="https://user-images.githubusercontent.com/91794127/145272127-0f2cc8c9-4b18-401f-8d58-0653b70c52db.png">

I suppose you are talking about this, it is set as such and the limits seems to be fairly similar on what I have on marlin

**Reply by supermerill** [reply]
Can you share a cimple project file where it's too noisy?
I can't see how this slicer can make your printer noisier, unless something is set (custom gcode?) to overwrite acceleration/jerk.

**Reply by lexCrr** [reply]
[Tree Facets sample.zip](https://github.com/supermerill/SuperSlicer/files/7682593/Tree.Facets.sample.zip)

I'm sorry I had to upload it as a zip. The model should be a vase mode christmas tree
